app/src/pages/inside/common/launchFiltersContainer/launchFiltersContainer.jsx

```diff
@@ -98,10 +109,47 @@ export class LaunchFiltersContainer extends Component {
     resetLocalSorting: () => {},
   };
 
+  state = {
+    isFilterTracked: false,
+  };
+
+  componentDidUpdate(prevProps) {
+    const { launchesFiltersReady, launchFilters } = this.props;
+
+    if (launchesFiltersReady) {
+      if (launchFilters !== prevProps.launchFilters || !this.state.isFilterTracked) {
```

Comment:
You shouldn't compare all launchFilters obj, because tracking will be invoke everytime filter condition is change, better check only filters length to avoid it.